src/styles/blocks/section1.scss

```diff
@@ -0,0 +1,37 @@
+.section_1 {
+  margin: 0;
+  padding: 32px 16px;
+  height: 134px;
+  width: 166px;
+  text-align: center;
+}
+
+.section_1_img {
+  height: 134px;
+  width: 160px;
+  margin-bottom: 36px;
+}
+
+.section_1__text {
+  height: 36px;
+  width: 166px;
+  font-size: 12px;
+  font-weight: 500;
+  top: 175px;
+  margin: 0;
+  color: #060b35;
+  line-height: 18px;
+  font-family: Roboto, sans-serif;
+  text-align: left;
+}
```

Comment:
The class name '.section_1__text' seems to follow the BEM naming convention, but make sure that 'section_1' is the block name and 'text' is the element name. If 'section_1' is not the block name, you should correct it to follow the BEM structure.